tests/e2e/plugin_workflow.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
I think the path to this file also has to be included in `tests/test_notebooks.py`